sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc.inc

```diff
@@ -428,6 +428,12 @@ function sync_package_lcdproc() {
 				$config_text .= "Device={$realport}\n";
 				$config_text .= "Size={$lcdproc_config['size']}\n";
 				break;
+			case "rawserial":
+                                $config_text .= "[{$lcdproc_config['driver']}]\n";
+                                $config_text .= "Device={$realport}\n";
+                                $config_text .= "Size={$lcdproc_config['size']}\n";
+                                $config_text .= "UpdateRate=1\n";
+                                break;
```

Comment:
Just a minor fix required, use TAB to indent. It's the project style

Reply:
I have changes the spaces to tabs and bumped the portversion in the make file as requested.
Sorry for that; this is my first pull request :)